src/main/java/core/basesyntax/service/implementation/ReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.implementation;
+
+import core.basesyntax.service.Reader;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderImpl implements Reader {
+    @Override
+    public List<String> readFromFile(String filePath) {
+        List<String> lines = new ArrayList<>();
+        File file = new File(filePath);
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String value = reader.readLine();
+            while (value != null) {
+                lines.add(value);
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file", e);
+        }
+        boolean remove = lines.remove(lines.get(0));
```

Comment:
rename your class to CsvReaderImpl
it always removes the first line - this is correct only for csv files